src/root/index.tsx

```diff
@@ -35,6 +42,12 @@ const store = createStore(
 	),
 );
 
+store.subscribe(throttle(() => {
+	saveState({
+		dashboard: store.getState().dashboard,
```

Comment:
i think we must create a configuration file outside this, because it's grow later.

and i don't like that way of subscription ,maybe we try to use a enhancer way?